From 9869458297495c900aeabff8574bf2afdf507988 Mon Sep 17 00:00:00 2001
From: Steven Rostedt <srostedt@redhat.com>
Date: Tue, 29 Jun 2010 12:29:21 -0400
Subject: [PATCH 3/7] trace-cmd: Update documentation for added '-i' option to
 trace-cmd record

Signed-off-by: Steven Rostedt <rostedt@goodmis.org>

Conflicts:

	Documentation/trace-cmd-record.1.txt
	trace-usage.c

Signed-off-by: John Kacur <jkacur@redhat.com>
---
 Documentation/trace-cmd-record.1.txt | 5 +++++
 trace-usage.c                        | 1 +
 2 files changed, 6 insertions(+)

diff --git a/Documentation/trace-cmd-record.1.txt b/Documentation/trace-cmd-record.1.txt
index 81e37ac..1f3b6f6 100644
--- a/Documentation/trace-cmd-record.1.txt
+++ b/Documentation/trace-cmd-record.1.txt
@@ -138,6 +138,11 @@ OPTIONS
     inside the kernel. Using "-b 10000" on a machine with 4 CPUs will make
     Ftrace have a total buffer size of 40 Megs.
 
+*-i*::
+    By default, if an event is listed that trace-cmd does not find, it
+    will exit with an error. This option will just ignore events that are
+    listed on the command line but are not found on the system.
+
 *-N* 'host:port'::
     If another machine is running "trace-cmd listen", this option is used to
     have the data sent to that machine with UDP packets. Instead of writing
diff --git a/trace-usage.c b/trace-usage.c
index 9df19eb..c8397af 100644
--- a/trace-usage.c
+++ b/trace-usage.c
@@ -36,6 +36,7 @@ static struct usage_help usage_help[] = {
 		"          -N host:port to connect to (see listen)\n"
 		"          -t used with -N, forces use of tcp in live trace\n"
 		"          -b change kernel buffersize (in kilobytes per CPU)\n"
+		"          -i do not fail if an event is not found\n"
 	},
 	{
 		"start",
-- 
1.7.11.7

